core/src/main/java/site/ycsb/ClientThread.java

```diff
@@ -117,26 +140,44 @@ public void run() {
       if (dotransactions) {
         long startTimeNanos = System.nanoTime();
 
-        while (((opcount == 0) || (opsdone < opcount)) && !workload.isStopRequested()) {
+        while (((totalopscount == 0) || (totalopsdone < totalopscount)) && !workload.isStopRequested()) {
 
           if (!workload.doTransaction(db, workloadstate)) {
             break;
           }
 
-          opsdone++;
+          if (isWarmUpDone()) {
+            if (payloadStart == null) {
+              payloadStart = System.currentTimeMillis();
+            }
+            opsdone++;
+            totalopsdone++;
+          } else {
+            warmupopsdone++;
+            totalopsdone++;
```

Comment:
Looks like we could move `totalopsdone++` out from `if/else` block. It's executed in both branches.